src/util/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required(VERSION 3.0)
+
+add_library(UtilLib INTERFACE)
```

Comment:
we forgot to mention `statusor.h` here

Reply:
Cmake says: "add_library INTERFACE library requires no source arguments."